deepchem/feat/tests/test_one_hot_featurizer.py

```diff
@@ -11,9 +11,23 @@ class TestOneHotFeaturizert(unittest.TestCase):
   Test OneHotFeaturizer.
   """
 
-  def test_onehot_featurizer(self):
+  def test_onehot_featurizer_arbitrary(self):
     """
-    Test simple one hot encoding.
+    Test simple one hot encoding for arbitrary string.
+    """
+    string = "abcdefghijklmnopqrstuvwxyzwebhasw"
+    charset = "abcdefghijklmnopqrstuvwxyz"
+    length = len(charset) + 1
+    featurizer = OneHotFeaturizer(charset)
+    feature = featurizer([string])  # Implicit call to featurize()
+    assert feature.shape == (1, 100, length)
```

Comment:
Can you set the max-length explicitly? Otherwise not clear where the 100 comes from

Reply:
This unit test was written this way to reflect the original test_one_hot_featurizer() method for SMILES.

test_one_hot_featurizer_with_max_length() explicitly assigns a max_length, while this method is not supposed to modify the max_length from the default. 

I'm happy to change that if you'd like.